### PR TITLE
SAK-29335 - New morpheus structure and skin with maven support to com…

### PR DESCRIPTION
**Pull request by SedueRey**
…pile

it

### CODE REVIEW

**Comment by buckett** [comment]
Are the images from an existing collections? Is there license compatible with Sakai?

**Comment by SedueRey** [comment]
Yes, they are Creative commons. It would be a test, we can modify them later and make our own to Sakai. This pull shows strenght of compass and maven to automatize this work.

File: reference/library/pom.xml [hunk 1]
```diff
@@ -60,9 +60,175 @@
         </dependency>
       </dependencies>
     </profile>
+    <profile>
```
Comment (buckett) [inline reference/library/pom.xml]: Would you be able to fix the indentation in this new section as it's a little all over the place.

**Comment by SedueRey** [comment]
No problem. I'll fix it

**Comment by buckett** [comment]
Did you look at using this maven plugin at all for doing the the compass compilation?

https://github.com/GeoDienstenCentrum/sass-maven-plugin

I don't have any experience of it, but was wondering if you had.

**Comment by steveswinsburg** [comment]
There was a legitimate concern raised on list about the removal of the icons. Could there be an update posted about that and how the new work is going to impact/cater for it?

> A question on the removal of tool icons: Have you left the hooks (i.e., CSS classes on navigation items / references inside tools) for the icons?

> I’ve heard wise UX folk warn against having too many icons in an interface. In most websites, with fairly static navigation options (i.e., a site having the same navigation options, with the same names, and in the same order each time the user visits the site) removing icons seems like the way to go, and that does seem to be the trend.

> However, in chatting with my colleagues on this, we believe there are some important characteristics of course/project sites in Sakai that make the tool icons very useful:

> **1. Instructors can add multiple instances of the Lessons, Web Content, and News tools...and these tools are designed to be renamed.** So, suppose an instructor adds multiple instances of the Lessons tool to a site and names them “Battle of the Somme,” “Russian Revolution,” and “Tanks!.” Without the Lessons tool icon, it’s unclear what tools these actually are until the user clicks on them. Imagine if the instructor also adds multiple instances of the Web Content tool or News tool to the same site. This could get pretty confusing, both for the student and the instructor--I’m imagining quite a bit of clicking around to figure out what’s what.

> **2. Instructors can (and do) rename tools, even those that do not allow for multiple instances.** We have seen instructors change Resources to “Course Readings” or “Documents”... or much stranger things. Without the recognizable Resources icon, it could get frustrating for a student who just wants to know where the courses files are.  

> **3. Different instructors add different tools, different numbers of tools, and in different orders.** In short, across a student’s course sites, the navigation can be consistently inconsistent. This can make it difficult for users to adapt and develop an expectation of where things will be. We believe the icons may serve as a consistent touchpoint for students across their different sites to counterbalance that variability. (Of course, we think that that variability and flexibility is ultimately a good thing.)

> In addition to the reasons above, in the new Gradebook we’re referencing tool icons within gradebook items that pull from another tool (e.g., Assignments, T&Q.).

> To help the left menu feel less cluttered in our own instance, we’re using monochromatic icons, which have seemed to go over pretty well here: https://www.evernote.com/l/AI1xqn0q9uhMLIvMzj6HcS5XN_kf2vFwb1oB/image.png

> Anyway, that’s what we’re thinking. Interested in hearing some other thoughts on this.

**Comment by juanjmerono** [comment]
**Matthew**, I've helped Eduardo with the maven stuff, he usually compiles scss with a compass client. I don't have any experience with any sass plugin, so I decided to use a generic ruby plugin to be able to control compass version independently from the plugin version. Anyway we can move to the plugin you've mentioned if you think is a better option.

**Steve**, We've taken care about Jeff's concerns (please take a look at the email sent to the core-team list 'Morpheus updates ready to go!') and that's why we compiled two versions of the skin, with and without icons. Now, It is pretty easy to change the default compiled skin to have icons (just one variable). If there is consensus in using icons in the default skin, we (or anyone) could change it very easily.

**Comment by SedueRey** [comment]
**Buckett**, 
 - I've formatted all *pom.xml* lines because it had a mix of tabs and spaces so I have thought to fix the entire file.
 - I have add the link of the set of icons in the *README* file but I plan to update those icons remains and make some updates by my own.

**Steve** as Juanjo said, there is two versions of the skin, with and without icons so we can test if our users get frustrated using one or the other. If the consensus is that icons are better we can change the default skin. I think this is a legitimate concert too and I want to test ASAP if our users prefer one version or the other. I don't want them to get lost using new interface.

**Comment by steveswinsburg** [comment]
Ok thanks guys!

**Pull request merged**